review-tools/gitaddrev

```diff
@@ -108,41 +111,19 @@ foreach (@ARGV) {
 }
 
 my @commit_message = map { (my $x = $_) =~ s|\R$||; $x } <STDIN>;
-my $author = $ENV{GIT_AUTHOR_EMAIL};
 my $trivial = !! grep(/^CLA:\s*Trivial\s*$/i, @commit_message);
 
-# If the author is a registered committer, that identity passes as a reviewer
-# too.  There is a twist, though...  see next comment
-if (my $rev = try_add_reviewer($author)) {
-
-    # So here's the deal: We added the commit author because we need to keep
-    # count of the total amount of reviewers, which includes the commit author
-    # if it's a registered committer.  However, the author can "reviewed
-    # themselves", so no Reviewed-by: should be added for that identity.
-    # However, we still need to check the @reviewers count below, so the
-    # solution is to record this rev tag separately and remove it from
-    # @reviewers after the count check.
-    $skip_reviewer = $rev;
-
-} else {
-
-    # In case the author is unknown to our databases or is lacking a CLA,
-    # we need to be extra careful to check if this is supposed to be a
-    # trivial commit.
-
-    # Note: it really should be enough to check if $author is unknown, since
-    # the databases are supposed to be consistent with each other.  However,
-    # let's be cautious and check both, in case someone has been registered
-    # as a known identity without having a CLA in place.
-    die "Commit author ",$author," has no CLA, and this is a non-trivial commit\n"
-	if !$trivial && grep { $_ eq $author } (@nocla_reviewers);
-
-    # Now that that's cleared, remove the author from anything that could cause
-    # more unnecessary errors (false positives).
-    @nocla_reviewers = grep { $_ ne $author } @nocla_reviewers;
-    @unknown_reviewers = grep { $_ ne $author } @unknown_reviewers;
-    $skip_reviewer = $author;
-}
+# Note: it really should be enough to check if $author is unknown, since
+# the databases are supposed to be consistent with each other.  However,
+# let's be cautious and check both, in case someone has been registered
+# as a known identity without having a CLA in place.
+die "Commit author ",$author," has no CLA, and this is a non-trivial commit\n"
+if !$trivial && grep { $_ eq $author } (@nocla_reviewers);
```

Comment:
This line should be indented 4 spaces (it should always have been, it was sloppy coding not to).  Also, the parenthesis is unnecessary, and frankly a bit confusing...